recipe/recipe.yaml

```diff
@@ -23,7 +23,7 @@ source:
       # backport (relevant parts of) https://github.com/triton-lang/triton/pull/7695
       - patches/0006-Add-python-3.14-wheel-build-7695.patch
```

Comment:
Surprised that this was not a patch application hazard. I rebased the patches locally, and this one is now obsolete.

Reply:
Yeah, `rattler-build` seems not to care that it's applied already. I've removed it locally already but don't want to restart jobs over it.